notebooks/profiling.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
```

Comment:
XiPandas is the old xi class - I didn't add the module again but I had it locally when I was testing. If anyone wants to reproduce the code, they should change paths to the test dataset and the initial xi code from the commit here https://github.com/czbiohub/pyxi/commit/448332e54007ce841a871b9ae8651e8e32c8a63b and name the `xi` class to `XiPandas`